lib/generators/react_on_rails/templates/client/README.md

```diff
@@ -0,0 +1,97 @@
+Example NPM Package
+===========================
+We've included an example package.json from https://github.com/shakacode/react-webpack-rails-tutorial which should get you started with your React project.
+
+Starting the node.js server:
+```
+npm start
+```
+
+Building client javascript files for production:
+```
+npm run build:client
+```
+
+Building server javascript files for production:
+```
+npm run build:server
+```
+
+Building client javascript files for development:
+```
+npm run build:dev:client
+```
+
+Building server javascript files for development:
```

Comment:
this is confusing

this is just for server rendering

we'll want to revisit this file

we can get some ideas from the tutorial 

this should be enough information that a newby knows how to do JS development the webpack+rails way and not the rails standard way